daily-challenges/2024-10/02/hunghoang3110.spec.ts

```
@@ -0,0 +1,13 @@
+// Javascript
+
+function findSecondLargest(array) {
+    const uniqueArray = [];
+    for (let i = 0; i < array.length; i++) {
+        if (uniqueArray.indexOf(array[i]) === -1) {
+            uniqueArray.push(array[i]);
+        }
+    }
+    let points = uniqueArray.toSorted(function (a, b) { return a - b });
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'points' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)